github4s/shared/src/main/scala/github4s/api/PullRequests.scala

```diff
@@ -28,87 +28,94 @@ import scala.language.higherKinds
 
 /** Factory to encapsulate calls related to PullRequests operations  */
 class PullRequests[C, M[_]](
-    implicit urls: GithubApiUrls,
-    C: Capture[M],
-    httpClientImpl: HttpRequestBuilderExtension[C, M]) {
+  implicit urls: GithubApiUrls,
+  C: Capture[M],
+  httpClientImpl: HttpRequestBuilderExtension[C, M]) {
 
   import Decoders._
   import Encoders._
 
   val httpClient = new HttpClient[C, M]
 
   /**
-   * List pull requests for a repository
-   *
-   * @param accessToken to identify the authenticated user
-   * @param headers optional user headers to include in the request
-   * @param owner of the repo
-   * @param repo name of the repo
-   * @param filters define the filter list. Options are:
-   *   - state: Either `open`, `closed`, or `all` to filter by state. Default: `open`
-   *   - head: Filter pulls by head user and branch name in the format of `user:ref-name`.
-   *     Example: `github:new-script-format`.
-   *   - base: Filter pulls by base branch name. Example: `gh-pages`.
-   *   - sort: What to sort results by. Can be either `created`, `updated`, `popularity` (comment count)
-   *     or `long-running` (age, filtering by pulls updated in the last month). Default: `created`
-   *   - direction: The direction of the sort. Can be either `asc` or `desc`.
-   *     Default: `desc` when sort is created or sort is not specified, otherwise `asc`.
-   * @return a GHResponse with the pull request list.
-   */
+    * List pull requests for a repository
+    *
+    * @param accessToken to identify the authenticated user
+    * @param headers optional user headers to include in the request
+    * @param owner of the repo
+    * @param repo name of the repo
+    * @param filters define the filter list. Options are:
+    *   - state: Either `open`, `closed`, or `all` to filter by state. Default: `open`
+    *   - head: Filter pulls by head user and branch name in the format of `user:ref-name`.
+    *     Example: `github:new-script-format`.
+    *   - base: Filter pulls by base branch name. Example: `gh-pages`.
+    *   - sort: What to sort results by. Can be either `created`, `updated`, `popularity` (comment count)
+    *     or `long-running` (age, filtering by pulls updated in the last month). Default: `created`
+    *   - direction: The direction of the sort. Can be either `asc` or `desc`.
+    *     Default: `desc` when sort is created or sort is not specified, otherwise `asc`.
```

Comment:
Could you please add the parameter description here?

Reply:
Done.